scripts/create-restore-job.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+import json, random, string, sys
```

Comment:
can you add some comments above on how the restore job works? e.g. use it against this helm release, etc

Reply:
there will be docs, soon...